packages/uikit-workshop/src/scripts/components/pl-nav/nav-dropdown.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../../sass/scss/core.scss';
+
+/**
+ * Nav Dropdown
+ * 1) On larger screens, display as dropdowns that
+ *    hangs over the header
+ */
+ .pl-c-nav__dropdown {
+  @media all and (min-width: $pl-bp-med) {
+    position: absolute;
+    top: 100%; /* 1 */
+    left: 0;
+    min-width: 12rem;
+    border-radius: $pl-border-radius-med;
+    border-style: solid;
+    border-width: 1px;
+    box-shadow: 0 2px 5px rgba(0,0,0,.1);
```

Comment:
Is there a benefit to using `rgba(0,0,0,.1)` vs `rgba($pl-color-black,.1)` here?